### ISSUE TITLE
Problem with setting up devlopment setup

### ISSUE THREAD

**Issue by manu-chroma** [cltk/cltk_frontend#50]
@kylepjohnson I cloned this repository and ran according to the README
 `meteor` command.
The server is running but no sample data is visible on the server. Is it related to the wrong setting up of `cltk_api.py` from https://github.com/cltk/cltk_api . Can someone else confirm the same. (Do update the repo to the latest commits before testing) Thanks.
Screenshots:
1. ![image](https://cloud.githubusercontent.com/assets/12952080/13752493/f366857e-ea34-11e5-88d4-094b73921b5b.png)
2. ![image](https://cloud.githubusercontent.com/assets/12952080/13752537/2a171516-ea35-11e5-92b5-5b922f13a4ef.png)
`OS:Ubuntu 15.10 (64-bit)`

**Comment by lukehollis** [comment]
No text is appearing due to the fact that the reading views need to be restructured as this ticket details: https://github.com/cltk/cltk_frontend/issues/44 and this ticket goes into more depth on: https://github.com/cltk/cltk_frontend/issues/43#issuecomment-196093131

Also, the text sync currently queries the text server in parallel instead of sequence, so you will need to restart your meteor server a few times before everything is ingested.  This is something that we need to resolve if you're interested in working on it--I've included the alternative proposed workflow here: https://github.com/cltk/cltk_frontend/blob/master/server/text-server-sync.js#L319

Leaving this issue open for now in case anyone else has similar problems.

**Comment by lukehollis** [comment]
New issue here: https://github.com/cltk/cltk_frontend/issues/51

**Comment by manu-chroma** [comment]
@lukehollis Can you suggest any temporary workaround for the time being so that I can fix https://github.com/cltk/cltk_frontend/issues/38 ?

**Comment by lukehollis** [comment]
For working on #38, I can currently see the Translation/Commentary panel functioning normally with demo content.  I had to restart the server a few times to sync content as I mentioned.  When you sync text from the text server, you will be able to see the translation panel as expected on the reading interface. If it does not appear at first, please refresh the page.

For example, this is what I'm currently seeing at http://localhost:3000/works/poemata/1
<img width="1187" alt="screen shot 2016-03-14 at 14 51 43" src="https://cloud.githubusercontent.com/assets/2633382/13756131/cdc1d20a-e9f4-11e5-9ad2-b32f26472520.png">

**Comment by manu-chroma** [comment]
@lukehollis It worked. Thanks.

**Comment by lukehollis** [comment]
No problem, and thanks for your contributions!

**Comment by manu-chroma** [comment]
@lukehollis Do you use `meteor` or `sudo meteor` ?
The way my mongodb is set up, I require `sudo` to run the server.

**Comment by lukehollis** [comment]
Hrm, no I haven't encountered that issue yet. What error messages are you seeing if you don't sudo?

I imagine you've probably come across it already, but does this from SO help at all? http://stackoverflow.com/questions/27558740/meteor-will-not-run-without-sudo#answer-27559623

**Comment by lukehollis** [comment]
The most recent commit restructuring reading views should resolve your issue with not seeing any text appearing.  Router errors persist, but I'm not too worried about them until we get further with either Flow or Redux.  Seems like Redux is the way to go for the reading environment.  

This is what I'm seeing now:
<img width="1377" alt="screen shot 2016-03-15 at 04 00 51" src="https://cloud.githubusercontent.com/assets/2633382/13771223/c05dab34-ea62-11e5-888f-fd5e3d382bc4.png">
